app/javascript/controllers/date_controller.js

```diff
@@ -0,0 +1,67 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["preselected"]
+
+  connect() {
```

Comment:
calendars are re-rendered every time the generator makes a change that triggers a turbo refresh, so i check to see which dates have been pre-selected on `connect()`